src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx

```diff
@@ -16,11 +14,11 @@ export default function CallToActionAlert({
 }) {
   const buttonClass =
     status === 'continue' ? 'va-button-primary' : 'usa-button-primary';
-
-  const alertProps = {
-    headline: heading,
-    level: headerLevel,
-    content: (
+  const CustomHeaderLevel = `h${headerLevel}`;
+  return (
+    // TODO fix this
```

Comment:
Something still need to be fixed here?

Reply:
Removed, just forgot to clean up the comment